### PR TITLE
Fix Add Task Function

### PR DESCRIPTION
**Pull request by hoang11549**
Fix Add Task Function and li18

### CODE REVIEW

**Comment by hoang11549** [comment]
ready

File: app/Http/Controllers/tasksControllers.php [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        //
+        $validator = new ValidatorTask();
```
Comment (quoctd-0607) [inline app/Http/Controllers/tasksControllers.php]: validate viết sang Request e

**Comment by hoang11549** [comment]
ready

**Comment by quoctd-0607** [comment]
merged

**Pull request merged**